src/main/scala/it/pagopa/interop/catalogmanagement/model/persistence/projection/EServiceCqrsProjection.scala

```diff
@@ -42,24 +43,27 @@ object EServiceCqrsProjection {
         ),
         Updates.set("data.descriptors.$[elem]", d.toDocument)
       )
-    case CatalogItemDeleted(cId)                               =>
+    case CatalogItemDeleted(cId)                                           =>
       Action(collection.deleteOne(Filters.eq("data.id", cId)))
-    case CatalogItemWithDescriptorsDeleted(c, dId)             =>
+    case CatalogItemWithDescriptorsDeleted(c, dId)                         =>
       ActionWithBson(
         collection.updateOne(Filters.eq("data.id", c.id.toString), _),
         Updates.pull("data.descriptors", Document(s"{ id : \"$dId\" }"))
       )
-    case CatalogItemDocumentAdded(esId, dId, doc, isInterface) =>
-      if (isInterface)
```

Comment:
you can remove brackets here and [here](https://github.com/pagopa/interop-be-catalog-management/pull/211/files?diff=split&w=1#diff-47a816ca70013b7d46ccce1ded9ea424047598057d62b9796b9120bf27023b79R66)